service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/RespondentController.java

```diff
@@ -68,35 +65,22 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
     private List<String> validate(CaseDetails caseDetails) {
         ImmutableList.Builder<String> errors = ImmutableList.builder();
 
-        Map<String, Object> respondentsData =
-            (Map<String, Object>) defaultIfNull(caseDetails.getData().get("respondents"), null);
-
         if (caseDetails.getData().containsKey("respondents1")) {
```

Comment:
I don't like we removed else statement, but still checking for the respondents1.

What if respondents1 is not there? Should we fail? 
If we are sure all cases are migrated, do we need this if statement?

Reply:
I have removed the if. When this callback is fired respondents1 will always be populated